components/toast/index.css

```
@@ -156,45 +150,34 @@ governing permissions and limitations under the License.
   flex: 0 0 auto;
   align-items: flex-start;
 
-  margin-block-start: var(--mod-toast-spacing-top-edge-to-divider, var(--spectrum-toast-spacing-top-edge-to-divider));
-  margin-block-end: var(--mod-toast-spacing-bottom-edge-to-divider, var(--spectrum-toast-spacing-bottom-edge-to-divider));
-
-  /* Divider color for Text | Action Button */
-  border-inline-start-color: var(--mod-toast-divider-color, var(--spectrum-toast-divider-color));
-  padding-inline-end: var(--mod-toast-spacing-close-button, var(--spectrum-toast-spacing-close-button));
+  margin-block-start: var(--mod-toast-spacing-top-edge-to-close-button, var(--spectrum-toast-spacing-top-edge-to-close-button));
+  margin-block-end: var(--mod-toast-spacing-bottom-edge-to-close-button, var(--spectrum-toast-spacing-bottom-edge-to-close-button));
+  padding-inline-start: var(--mod-toast-spacing-text-to-close-button, var(--spectrum-toast-spacing-text-to-close-button));
+  padding-inline-end: var(--mod-toast-spacing-close-button-to-edge, var(--spectrum-toast-spacing-close-button-to-edge));
 
   .spectrum-CloseButton {
     align-self: flex-start;
   }
 }
 
+.spectrum-Toast:has(.spectrum-Button) {
+  .spectrum-Button {
```

```suggestion
.spectrum-Toast:has(.spectrum-Button) {
  .spectrum-Button {
```
Could this first selector be written without the `:has`? `.spectrum-Toast .spectrum-Button`?